packages/rulesets/src/native/utilities/swagger-workspace.ts

```diff
@@ -83,18 +82,25 @@ export namespace Workspace {
     if (!source || !source.value) {
       return []
     }
-    let result:{[key:string]:EnhancedSchema} = {}
-    const model = source.value
+    let result: { [key: string]: EnhancedSchema } = {}
+    let model = source.value
+    // for list call, the properties are in the value.items.$ref
+    if (model.properties && model.properties.value && model.properties.value.items && model.properties.value.items.$ref) {
```

Comment:
I think you can simplify it like this
```suggestion
    if (model?.properties?.value?.items?.$ref) {
```

Reply:
Ah good point.
thanks.